docs/tutorials/prometheus_locked_down.md

```diff
@@ -0,0 +1,1418 @@
+    - "acceptorConfigurations.\"amqps\".params.port=61617"
+    - "acceptorConfigurations.\"amqps\".params.protocols=amqp"
+    - "acceptorConfigurations.\"amqps\".params.securityDomain=activemq"
+    - "acceptorConfigurations.\"amqps\".params.sslEnabled=true"
+    - "acceptorConfigurations.\"amqps\".params.needClientAuth=true"
+    - "acceptorConfigurations.\"amqps\".params.saslMechanisms=EXTERNAL"
+    - "acceptorConfigurations.\"amqps\".params.keyStoreType=PEMCFG"
+    - "acceptorConfigurations.\"amqps\".params.keyStorePath=/amq/extra/secrets/amqps-pem/_amqps.pemcfg"
+    - "acceptorConfigurations.\"amqps\".params.trustStoreType=PEMCA"
+    - "acceptorConfigurations.\"amqps\".params.trustStorePath=/amq/extra/secrets/activemq-artemis-manager-ca/ca.pem"
+  deploymentPlan:
+    extraMounts:
+      secrets: [artemis-broker-jaas-config, amqps-pem]
+EOF
+```
+```shell markdown_runner
+activemqartemis.broker.amq.io/artemis-broker created
+```
+
+Wait for the broker to be ready.
+
+```{"stage":"deploy"}
+kubectl wait ActiveMQArtemis artemis-broker --for=condition=Ready --namespace=locked-down-broker --timeout=300s
+```
+```shell markdown_runner
+activemqartemis.broker.amq.io/artemis-broker condition met
+```
+
+## Scrape the broker
+
+### Configure and Deploy Prometheus
+
+With the broker running, the next step is to configure Prometheus to scrape its
+metrics endpoint. This section uses the Prometheus Operator installed earlier to
+manage a `Prometheus` instance. The instance is configured via a
+`ServiceMonitor` to securely scrape the broker's metrics endpoint using the mTLS
+certificates created in the [Broker and Client
+Certificates](#create-broker-and-client-certificates) section.
+
+#### Create the ServiceMonitor and Prometheus Instance
+
+**ServiceMonitor** is a Kubernetes custom resource that tells the Prometheus Operator how to scrape metrics from services. It defines the scraping configuration including:
+
+1. **Which service to scrape** (using label selectors)
+2. **How to connect securely** (HTTPS with mTLS certificates)
+3. **Authentication details** (which certificates to use)
+
+**Prometheus Resource** tells the Prometheus Operator to deploy a Prometheus server that will:
+- Automatically discover ServiceMonitors with matching labels
+- Use the specified service account for Kubernetes API access
+- Store and query the collected metrics
+
+Using the Prometheus Operator simplifies deployment by managing the complex Prometheus configuration automatically.
+
+First, create the `ServiceMonitor` that tells Prometheus how to scrape the broker's metrics endpoint with mTLS authentication:
+
+```{"stage":"scrape", "runtime":"bash", "label":"create servicemonitor"}
+kubectl apply -f - <<EOF
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: artemis-broker-monitor
+  namespace: locked-down-broker
+  labels:
+    # This label is used by the Prometheus resource to discover this monitor.
+    release: prometheus
```

Comment:
Can we use a label also more related to artemis?

Reply:
What label do you have in mind?

Reply:
This is because the Prometheus instance created by the `Prometheus` CR is not a datasource of the Grafana instance, see
https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/values.yaml#L1447
https://docs.solo.io/gloo-mesh-enterprise/latest/observability/tools/grafana/setup/

Reply:
Thanks @brusdev, it seems like it's going the right direction with your findings. Strangely enough, setting all of that I get a weird "christmas tree" effects on the grafana dashboard

<img width="3528" height="1013" alt="image" src="https://github.com/user-attachments/assets/e87975cb-3d42-417a-86cb-c6f34a905ab6" />

with diagrams getting in and out as refreshes pass by.

I need to get a deeper understanding of the situation, I haven't looked at the logs yet, but will report any findings here.